src/Behat/Config/ExtensionInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config;
+
+interface ExtensionInterface extends ConfigInterface
```

Comment:
As with #1547 I would prefer `ExtensionConfigInterface` - especially for this one, as I hope extension developers will implement it themselves to provide custom DTOs / helpers, so we want to make it really clear to them / in their codebase what it is.

Reply:
done